tests/declare-program/programs/declare-program/src/lib.rs

```diff
@@ -89,6 +93,270 @@ pub mod declare_program {
 
         Ok(())
     }
+
+    pub fn account_utils(_ctx: Context<Utils>) -> Result<()> {
```

Comment:
Looks like something went wrong during the rebase. These tests were recently moved out of here in https://github.com/solana-foundation/anchor/pull/4109.